README.md

```diff
@@ -1,8 +1,13 @@
 [![codecov](https://codecov.io/gh/opensearch-project/opensearch-sdk-java/branch/main/graph/badge.svg)](https://codecov.io/gh/opensearch-project/opensearch-sdk-java)
+[![GHA gradle check](https://github.com/opensearch-project/opensearch-sdk-java/actions/workflows/build.yml/badge.svg)](https://github.com/opensearch-project/opensearch-sdk-java/actions/workflows/build.yml)
+[![GHA validate pull request](https://github.com/opensearch-project/opensearch-sdk-java/actions/workflows/wrapper.yml/badge.svg)](https://github.com/opensearch-project/opensearch-sdk-java/actions/workflows/wrapper.yml)
+[![Code Hygiene](https://github.com/opensearch-project/opensearch-sdk-java/actions/workflows/code-hygiene.yml/badge.svg)](https://github.com/opensearch-project/opensearch-sdk-java/actions/workflows/code-hygiene.yml)
 
 # OpenSearch SDK
```

Comment:
This was missed in the "for Java" change. Can we fix it as long as we're touching this file for other reasons?  Also as long as we're editing this file, there seems a mismatch below in contributing header vs. developer guide link, but there's also a contributing link.